policy/modules/system/init.te

```diff
@@ -398,6 +408,7 @@ ifdef(`init_systemd',`
 	# If /etc/localtime is missing, a watch on /etc is added.
 	files_watch_etc_dirs(init_t)
 	files_watch_etc_symlinks(init_t)
+	files_dontaudit_write_var_dirs(init_t)
```

Comment:
Are you sure this is ok? I seem to remember this happening when systemd is attempting to determine if the location is writable.

Reply:
I'm not sure if this is actually required or not. In my testing with this dontaudited I haven't seen any problems, but I see no objections to allowing this access either.